### PR TITLE
fix: flaws in web/javascript batch 6

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by nschonni** [comment]
@wbamberg any issues with this one?

File: files/en-us/web/javascript/reference/global_objects/webassembly/runtimeerror/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 <h2 id="Constructor">Constructor</h2>
 
 <dl>
- <dt><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/RuntimeError/RuntimeError"><code>WebAssembly.RuntimeError()</code></a></dt>
+ <dt><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/RuntimeError/RunTimeError"><code>WebAssembly.RuntimeError()</code></a></dt>
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/webassembly/runtimeerror/index.html:19]: It looks like the slug for this page is wrong:

https://github.com/mdn/content/blob/d2d2314c40b95b5c408d8f27d704f299b4660d03/files/en-us/web/javascript/reference/global_objects/webassembly/runtimeerror/runtimeerror/index.html#L3

...should be "Web/JavaScript/Reference/Global_Objects/WebAssembly/RuntimeError/RuntimeError"

Probably better to fix it there?

**Review by wbamberg — CHANGES_REQUESTED** [review]
I think there's one place where the fix should be to correct the slug for the target of a link. Otherwise this looks great to me.

File: files/en-us/web/javascript/reference/global_objects/webassembly/runtimeerror/index.html (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/javascript/reference/global_objects/webassembly/runtimeerror/index.html:19]: Yeah, if the slug is fixed there, the next time a flaw run is done across the repo, it'll fix the occurrences

**Pull request closed without merging**